python/overrides.js

```diff
@@ -15,6 +15,7 @@
   
     'should waitBeforeCapture in open': { skipEmit: true },
     'should waitBeforeCapture in check': { skipEmit: true },
+    'AUTproxy should proxy resources test manually only': { skipEmit: true },
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8974587)
```suggestion
    "AUTproxy should proxy resources test manually only": {skipEmit: true},
```